.circleci/src/jobs/check_frontend.yml

```diff
@@ -29,13 +27,11 @@ steps:
       command: |
         fvm flutter analyze --fatal-infos --fatal-warnings
         fvm flutter analyze pubs/df_build_config --fatal-infos --fatal-warnings
-        fvm flutter analyze pubs/df_protobuf --fatal-infos --fatal-warnings
```

Comment:
:question: This check is now also done by the build runner?

Reply:
I completely removed `pubs/df_protobuf`